tests/e2e/utils/common.go

```diff
@@ -66,25 +65,40 @@ func GenerateDeploymentBody(deploymentName, image string, port, replica int) (er
 	return nil, temp
 }
 
-//Function to handle app deployment/delete deployment.
-func HandleDeployment(operation string, apiserver string, UID string, ImageUrl string, replica int) bool {
+//GeneratePodBody to generate Pod body
+func GeneratePodBody(podName, image, nodeselector string) (error, map[string]interface{}) {
```

Comment:
see previous comment for generating deployment object. we need to have a method to create pod object using apis and then marshal to json.

Reply:
@rohitsardesai83 , Removed the logic of json object creation and added the logic to create a deployment object and pod objects through the function itself